fw/beef.cpp

```diff
@@ -67,11 +67,22 @@ void debounce(DebounceState* debounce, uint16_t mask) {
     (debounce->debounce(button_state & mask));
 }
 
+void check_for_dfu() {
+  button_state = 0;
+  for (int i = 0; i < BUTTONS; ++i) {
+    process_button(buttons[i].INPUT_PORT.PIN,
+                   i,
+                   buttons[i].input_pin);
+  }
+  if ((button_state & BUTTON_1) &&
+      (button_state & BUTTON_2)) {
```

Comment:
This can be simplified to `button_state == (BUTTON_1 | BUTTON_2)`, or call `is_pressed(BUTTON_1 | BUTTON_2)`.